Vizzuality/heco-invest
# Fix front-end production vulnerabilities

This PR fixes two front-end vulnerabilities found in production by Dependabot with `minimist` and `glob-parent`.

I've also added instructions on how to fix future vulnerabilities in the README file.

## Testing instructions

Run `yarn npm audit --recursive --environment production` and make sure there are no vulnerabilities anymore.

## Tracking

Not tracked.
